README.md

```diff
@@ -54,21 +54,25 @@ environment, contribution workflow and submitting patches, etc.
 
 ## Learning Resources
 
-* [How we build TiDB](https://www.pingcap.com/blog/2016-10-17-how-we-build-tidb/)
-* [A Brief Introduction of TiDB](https://www.pingcap.com/blog/2017-05-23-perconalive17/)
-* [https://pingcap.com/docs](https://pingcap.com/docs)
-* [TiDB Internal (I) - Data Storage](https://pingcap.com/blog/2017-07-11-tidbinternal1/) or [三篇文章了解 TiDB 技术内幕 - 说存储](https://pingcap.com/blog-cn/tidb-internal-1/) in Chinese
-* [TiDB Internal (II) - Computing](https://pingcap.com/blog/2017-07-11-tidbinternal2/) or [三篇文章了解 TiDB 技术内幕 - 说计算](https://pingcap.com/blog-cn/tidb-internal-2/) in Chinese
-* [TiDB Internal (III) - Scheduling](https://pingcap.com/blog/2017-07-20-tidbinternal3/) or [三篇文章了解 TiDB 技术内幕 - 谈调度](https://pingcap.com/blog-cn/tidb-internal-3/) in Chinese
-* [TiDB 源码阅读系列文章](https://pingcap.com/blog-cn/#TiDB-%E6%BA%90%E7%A0%81%E9%98%85%E8%AF%BB), in Chinese
-* [TiKV 源码解析系列文章](https://pingcap.com/blog-cn/#TiKV-%E6%BA%90%E7%A0%81%E8%A7%A3%E6%9E%90), in Chinese
+Learning resources are collected in the [learning
+resources](./learning-resources/README.md) page. Here you can find all the
```

Comment:
we also have a chinese version here https://github.com/pingcap/presentations/blob/master/hackathon-2019/reference-document-of-hackathon-2019.md
should we merge them into one doc?

Reply:
Yeah. We can do it later in another PR?